docs/content/flags.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Global Flags"
 description: "Rclone Global Flags"
-date: "2020-02-01T10:26:53Z"
+date: "2020-02-12T07:00:07+01:00"
```

Comment:
?

Reply:
This automatically done when you generate docs.